extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -110,52 +119,68 @@ function onSubmit() {
-    Please give us an email address you want to use.  Once we confirm that it works,
-    you'll be asked to set a password and then you can start filing [% terms.bugs %] and helping fix them.
-  </li>
-  </ol>
+    <p>
+      Great!  There are four things to know and do:
+    </p>
+    <ol>
+    <li>
+      Please read our
+      <a href="https://developer.mozilla.org/en/Bug_writing_guidelines" target="_blank">[% terms.Bug %] Writing Guidelines</a>.
+    </li>
+    <li>
+      [% terms.Bugzilla %] is a public place. Your comments and other activities
+      on [% terms.bugs %] will generally be publicly visible, and your email address will be
+      accessible through public APIs and will be visible to all logged-in users
+      of [% terms.Bugzilla %]. Some people use an
+      <a href="https://www.lifewire.com/top-free-email-services-1171481"
+      target="_blank">alternative email address</a> for this reason.
+      See Mozilla's <a href="https://www.mozilla.org/privacy/websites/"
+      target="_blank">Websites, Communications & Cookies Privacy Notice</a> for
+      more information on our privacy policies.
+    </li>
+    <li>
+      When using [% terms.Bugzilla %] to submit patches, comments, code, and any other content,
+      you agree to our policies on open source licensing and content submission. See Mozilla’s
+      <a href="https://www.mozilla.org/about/legal/terms/mozilla/" target="_blank">Websites
+      and Communications Terms of Use</a> for more information.
+    </li>
+    <li>
+      You understand that your conduct on this site is subject to both <a href="page.cgi?id=etiquette.html">[% terms.Bugzilla %] Etiquette</a>, 
+      and the <a href="https://www.mozilla.org/about/governance/policies/participation/">Mozilla Community Participation Guidelines</a>. 
+      By creating an account, you agree to abide by them.
+    </li>
+    <li>
+      Please give us an email address you want to use. Once we confirm that it works,
+      you'll be asked to set a password and then you can start filing [% terms.bugs %] and helping fix them.
+    </li>
+    </ol>
   </div>
 
   <h2 class="column-header">Create an account</h2>
 
   <form method="post" action="createaccount.cgi" onsubmit="return onSubmit()">
-  <table id="create-account-form">
-  <tr>
+    <table id="create-account-form">
+    <tr>
     <td class="label">Email Address:</td>
     <td>
-      <input size="35" id="login" name="login" placeholder="you@example.com">[% Param('emailsuffix') FILTER html %]</td>
+      <input size="35" id="login" name="login" placeholder="you@example.com">[% Param('emailsuffix') FILTER html %]
+    </td>
+    </tr>
+    <tr>
+    <td>
+      I have read <a href="page.cgi?id=etiquette.html">[% terms.Bugzilla %] Etiquette</a>
+      and the <a href="https://www.mozilla.org/about/governance/policies/participation/">Mozilla Community Participation Guidelines</a>
+      and agree to abide by them.
+      <input type="checkbox" id="agree" value="agreed">
```

Comment:
"Agree" is probably not the best name for this - agree to what? Perhaps "etiquette" would be a better name.